cms/templates/library.html

```diff
@@ -58,12 +58,12 @@ <h3 class="sr">${_("Page Actions")}</h3>
             <ul>
                 <li class="nav-item">
                     <a href="#" class="button new-button new-component-button">
-                        <i class="icon fa fa-plus icon-inline"></i> <span class="action-button-text">${_("Add Component")}</span>
+                        <span class="icon fa fa-plus icon-inline" aria-hidden="true"></span> <span class="action-button-text">${_("Add Component")}</span>
                     </a>
                 </li>
                 <li class="action-item action-toggle-preview nav-item">
                     <a href="#" class="button button-toggle-preview action-button toggle-preview-button is-hidden">
-                        <i class="icon fa fa-refresh"></i>
+                        <span class="icon fa fa-refresh" aria-hidden="true"></span>
```

Comment:
It looks like this doesn't need sr text because the text is added via JS.

![image](https://cloud.githubusercontent.com/assets/484484/15759418/8ed0adb0-28dc-11e6-8dcf-a793967b55cd.png)